src/Idle.DataAccess/ModelBase.cs

```diff
@@ -0,0 +1,54 @@
+﻿using SQLite;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Idle.DataAccess
+{
+    /// <summary>
+    /// Base class for all models
+    /// </summary>
+    public abstract class ModelBase
+    {
+
+        /// <summary>
+        /// Primary Key for SQLite database
+        /// </summary>
+        [AutoIncrement]
+        public int ID { get; set; }
+
+
+
+        /// <summary>
+        /// An enum defining the different levels of difficulty
+        /// </summary>
+        public enum Difficulty
```

Comment:
I would not define this in any class as it does not belog to the type `ModelBase`.
i would define it outside the class in a separate file. I think you will find a good place as you did with the other classes 😉

Reply:
Yes, thats probably a lot better.